classes/base.lua

```diff
@@ -101,6 +102,15 @@ function class:declareOptions ()
     end
     return self._name
   end)
+  self:declareOption("orientation", function(_, orientation)
```

Comment:
Given other discussions and the suggestion that there are no other possible orientations of interest relative to the way this feature interacts with the papersize, I think we should make this option take a boolean value, `landscape=true` rather than `orientation=landscape`.